tools/tpm2_send.c

```diff
@@ -70,10 +70,10 @@ static int read_command_from_file(FILE *f, tpm2_command_header **c,
     LOG_INFO("command code: 0x%08x", tpm2_command_header_get_code(command));
 
     ret = fread(command->data, data_size, 1, f);
-    if (ret != 1 && ferror(f)) {
-        LOG_ERR("Failed to read command body: %s", strerror (errno));
-        free(command);
-        return -1;
+    if (ret != 1) {
+        LOG_ERR("Failed to read command body: %s", feof(f) ? "EOF" : strerror (errno));
+	free(command);
+	return -1;
```

Comment:
Looks like hard tabs slipped in.  Otherwise, LGTM.

Reply:
> Looks like hard tabs slipped in. Otherwise, LGTM.

new machine... didn't set up vim thnaks